docs/rest_api.md

```diff
@@ -59,7 +59,7 @@ You can also retrieve these credentials using the CLI running `wsk property get
 The following is an example that uses the [cURL](https://curl.haxx.se) command tool to get the list of all packages in the `whisk.system` namespace:
 
 ```bash
-curl -u USERNAME:PASSWORD https://openwhisk.ng.bluemix.net/api/v1/namespaces/whisk.system/packages
+curl -u USERNAME:PASSWORD https://APIHOST/api/v1/namespaces/whisk.system/packages
```

Comment:
```suggestion
curl -u USERNAME:PASSWORD https://$APIHOST/api/v1/namespaces/whisk.system/packages
```

Reply:
thanks - i used APIHOST without the $ explicitly here to match the USERNAME:PASSWORD --- and added a note later before the curl examples about $APIHOST.

i can change to {APIHOST} perhaps - the doc uses that earlier (and similarly replace U:P).